app/views/events/_nav.html.erb

```diff
@@ -57,7 +57,7 @@
       event_check_deposits_path(@event),
       tooltip: "Deposit checks",
       icon: "cheque",
-      selected: local_assigns[:selected] == :deposit_check if policy(@event.check_deposits.build).index? %>
+      selected: local_assigns[:selected] == :deposit_check if policy(@event.check_deposits.build).index? && organizer_signed_in?(as: :member) %>
```

Comment:
Since we're already calling the policy here, we might want to just include the member check inside the policy for `index?`